src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/generators/CoalGenerator.java

```diff
@@ -22,6 +24,11 @@ protected void registerDefaultFuelTypes() {
         registerFuel(new MachineFuel(12, new ItemStack(Material.BLAZE_ROD)));
         registerFuel(new MachineFuel(20, new ItemStack(Material.DRIED_KELP_BLOCK)));
 
+        // Boats
+        for (Material mat : Tag.ITEMS_BOATS.getValues()) {
+            registerFuel(new MachineFuel(6, new ItemStack(mat)));
```

Comment:
You use 5 planks to craft a boat, so it should be 5 here.

Reply:
i got this info straight from the wiki https://minecraft.gamepedia.com/Smelting im fine changing it if you actually want to have smelting time differ from mc official time

Reply:
I'd rather have them make sense than be accurate :LUL: